packages/pancake-uikit/src/components/Message/Message.tsx

```diff
@@ -24,13 +24,11 @@ const MessageContainer = styled.div<MessageProps>`
   })}
 `;
 
-const Message: React.FC<MessageProps> = ({ children, variant, ...props }) => {
+const Message: React.FC<MessageProps> = ({ children, variant, icon, ...props }) => {
   const Icon = Icons[variant];
   return (
     <MessageContainer variant={variant} {...props}>
-      <Box>
-        <Icon color={variants[variant].borderColor} width="24px" mr="12px" />
-      </Box>
+      <Box mr="12px">{icon ?? <Icon color={variants[variant].borderColor} width="24px" />}</Box>
```

Comment:
Do we still want a margin right of 12px if an icon is not passed in?

Reply:
Yes, if icon is not passed in it would behave exactly as it is now (margin just moved to the container and not directly on the icon). And if you pass custom icon margin is still there cause it's on the container, no need to define it manually.
And without margin icon is squished right next to the text, you can try it out in storybook.